Traceback (most recent call last):
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/usr/local/Cellar/python/3.6.5_1/Frameworks/Python.framework/Versions/3.6/lib/python3.6/asyncio/base_events.py", line 468, in run_until_complete
    return future.result()
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/client.py", line 536, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/chrislevy/Library/Caches/pypoetry/virtualenvs/teach-python-0grHqIjk-py3.6/lib/python3.6/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import random
i = random.randint(1, 10)

while True:
    # guess = input('Guess a number between 1 and 10. Type "stop" to exit.')
    # if guess == stop then break the loop and print a message saying all done
    # otherwise, guess should be a number. Remember you need to convert inputs to integers.
    # Check if the guess is the same as i or too high or too low and use break in proper place.
------------------

[0;36m  File [0;32m"<ipython-input-6-e3fda04fd94e>"[0;36m, line [0;32m8[0m
[0;31m    # Check if the guess is the same as i or too high or too low and use break in proper place.[0m
[0m                                                                                               ^[0m
[0;31mSyntaxError[0m[0;31m:[0m unexpected EOF while parsing

SyntaxError: unexpected EOF while parsing (<ipython-input-6-e3fda04fd94e>, line 8)

